Changelog.md

```diff
@@ -2,10 +2,11 @@
 
 Format for this file derived from [http://keepachangelog.com](http://keepachangelog.com).
 
-## Unreleased
+## 0.3.0 - 2022-10-07
```

Comment:
You should yank this release because it contains a ton of breaking changes and should be released as 1.0.0.  It also can't be released yet because it depends on the unreleased 1.0.0 ffi-gdal.

The only service running against develop ffi-gdal/ffi-gdal-extensions is logged_data.